tests/platform/test_reboot.py

```diff
@@ -117,3 +150,73 @@ def test_warm_reboot(testbed_devices, conn_graph_facts):
             pytest.skip("ISSU is not supported on this DUT, skip this test case")
 
     reboot_and_check(localhost, ans_host, conn_graph_facts["device_conn"], reboot_type="warm")
+
+
+@pytest.fixture(params=[15, 5])
+def power_off_delay(request):
+    """
+    used to parametrized test cases on power_off_delay
```

Comment:
General comment:
Please use the common function header for each function. 
It should include summary which describes what the function is all about and params input and output

There is an example in the same file
'''
    @summary: Check the reboot cause on DUT.
    @param dut: The AnsibleHost object of DUT.
'''